mlir/test/Dialect/Transform/test-pass-application.mlir

```diff
@@ -164,6 +164,97 @@ module attributes {transform.with_named_sequence} {
 
 // -----
 
+// CHECK-LABEL: func private @valid_multiple_params_as_list_option()
+module {
+  func.func @valid_multiple_params_as_list_option() {
```

Comment:
nit: I'd add one more function that's not excluded, otherwise it looks like the transform doesn't do anything at all whether it's intentional or by mistake

Reply:
Hmm. By the CHECK-LABEL, we have that a `func.func` went from public to private. By the other `CHECK`s we have that the thus named functions were properly excluded from being marked private.

Might you have missed that the "label" function already serves the purpose of showing the pass has an (intended) effect?